recipes/lief/all/conanfile.py

```diff
@@ -109,13 +122,32 @@ def generate(self):
         tc.variables["LIEF_DOC"] = False
         tc.variables["LIEF_LOGGING"] = False
         tc.variables["LIEF_PYTHON_API"] = False
+        if Version(self.version) >= "0.12.2":
+            tc.variables["LIEF_USE_CCACHE"] = False
+            tc.variables["LIEF_OPT_MBEDTLS_EXTERNAL"] = True
+            tc.variables["LIEF_OPT_NLOHMANN_JSON_EXTERNAL"] = True
+            tc.variables["LIEF_OPT_FROZEN_EXTERNAL"] = True
+            tc.variables["LIEF_OPT_UTFCPP_EXTERNAL"] = True
+            tc.variables["LIEF_EXTERNAL_SPDLOG"] = True
+            tc.variables["LIEF_OPT_EXTERNAL_LEAF"] = True
+            tc.variables["LIEF_OPT_EXTERNAL_SPAN"] = True
         tc.generate()
 
         deps = CMakeDeps(self)
         deps.generate()
 
-    def build(self):
+    def _patch_sources(self):
         apply_conandata_patches(self)
+
+        # dirty fix for compilation error on gcc 5
```

Comment:
This should be a patch and upstreamed 🙈

Reply:
@prince-chrismc 
Thanks.
I created PR for it.

Reply:
Can you please move this out of the recipe, with a `"patch_source"` field point to you PR 🙏 this way it will be easier to track and  bump version when this patch is accepted

Reply:
@prince-chrismc 
I see.
I am going to create additional patch for it.

Reply:
Fixed